.github/workflows/reliability_report.yml

```
@@ -9,27 +9,27 @@ name: Update CI reliability
 on:
   schedule:
     - cron: '0 0 * * *'
+  workflow_dispatch:
   
 jobs:
   create-report:
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/setup-node@v1
+    - uses: actions/setup-node@v3.0.0
```

```suggestion
    - uses: actions/setup-node@v3
```
https://github.com/actions/setup-node/issues/425